src/overlays/actors/ovl_Obj_Jgame_Light/z_obj_jgame_light.h

```diff
@@ -6,10 +6,25 @@
 struct ObjJgameLight;
 
 #define OBJJGAMELIGHT_GET_7F(thisx) ((thisx)->params & 0x7F)
+#define OBJLUPYGAMELIFT_HEALTH_1(thisx) ((thisx)->colChkInfo.health & 1)
+#define OBJLUPYGAMELIFT_HEALTH_2(thisx) ((thisx)->colChkInfo.health & 2)
+#define OBJLUPYGAMELIFT_HEALTH_4(thisx) ((thisx)->colChkInfo.health & 4)
+#define OBJLUPYGAMELIFT_HEALTH_8(thisx) ((thisx)->colChkInfo.health & 8)
```

Comment:
https://discord.com/channels/688807550715560050/688851976746041391/991357782559305818
It actually seems better to make individual defines for the flag values instead of macros.

For instance instead of using `OBJLUPYGAMELIFT_HEALTH_1(&this->actor)` in the `c` file, we will instead do 
`(this->actor.colChkInfo.Health & OBJLUPYGAMELIFT_IGNITE_FIRE)` with a 
`#define OBJLUPYGAMELIFT_IGNITE_FIRE (1 << 0)` in the `.h` file (since it will be used in another actor).
Note it looks like these defines should also be used with the `unk_1B8` variable, which can probably be renamed to `prevHealth`.

Reply:
Mm, should probably make the appropriate replacements in EnJgameTsn too.

Reply:
Yeah that too. Looks like the appropriate variable is `unk_208` in that actor.